reddit_service_websockets/socketserver.py

```diff
@@ -106,20 +111,48 @@ def start_response(self, status, headers, exc_info=None):
 
 
 class SocketServer(object):
-    def __init__(self, metrics, dispatcher, mac_secret, ping_interval):
+    def __init__(self, metrics,
+                 dispatcher,
+                 mac_secret,
+                 ping_interval,
+                 admin_auth,
+                 conn_shed_rate,
+    ):
         self.metrics = metrics
         self.dispatcher = dispatcher
         self.signer = MessageSigner(mac_secret)
         self.ping_interval = ping_interval
+        self.admin_auth = admin_auth
+        self.shed_rate_per_sec = conn_shed_rate
         self.status_publisher = None
+        self.quiesced = False
+        self.connections = set()
```

Comment:
List probably works better here actually; I think I initially chose to use set anticipating some stuff around checking connection existence in the set but things ended up being simpler. Updating!